pydarn/plotting/power.py

```diff
@@ -0,0 +1,149 @@
+#  Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+#  Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+from pydarn import SuperDARNRadars, exceptions, RTP
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_lag0
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
+
+    @classmethod
+    def plot_lag0(cls, records: List[dict], beam_num: int = 0,
+                  compare: bool = True, frequency: float = 11000,
+                  statistical_calc: object = np.mean):
+
+        """
+        This function will calculate and plot a statistic of the lag-0 
+        power of each record as a function of time.
+        
+        This function applies the statistical function (ex. numpy.mean) to the pwr0 vector
+        (lag-0 power for each range) for each record before plotting the results from all 
+        records chronologically. 
+
+        Extra  Details
+        --------------
+        If the records that are input correspond to politescan data then this
+        can be a useful method for determining background interference
+        variations and patterns.
```

Comment:
Just to clarify what `politescan` is, and how this code could be used more widely for studying interference when the radar is transmitting. Please check that I have the correct CPID :) 

```suggestion
This code can be used to study background interference in rawacf data when the radar has been operating in a receive-only mode such as "politescan" (cpid -3380), or during periods without any obvious coherent scatter returns from any range.
```

Reply:
CPID is correct!